resources/js/components/BuildSummary.vue

```diff
@@ -53,6 +53,16 @@
               </span>
             </td>
           </tr>
+          <tr>
+            <th>
+              Labels:
+            </th>
+            <td style="white-space: nowrap;">
+              <span v-if="cdash.build.labels.length > 0">
+                {{ cdash.build.labels.join(', ') }}
+              </span>
+            </td>
+          </tr>
```

Comment:
Should we move the `v-if="cdash.build.labels.length > 0"` so that the entire row will display only if the build has any labels associated with it? On the other hand, it might be confusing for users who are looking a build's labels if sometimes that information appears in the summary page but sometime it doesn't. (As opposed to seeing an empty cell next to `Labels:` which makes it clear that the build has no labels)

Reply:
I think it's reasonable to not show empty cells in the table.  We should only show the information available in my opinion.